src/site/_collections/authors.js

```diff
@@ -71,56 +71,68 @@ module.exports = (collections) => {
   /** @constant @type {Object.<string, Author>} @default */
   const authors = {};
 
-  Object.values(contributors)
-    .sort((a, b) => a.title.localeCompare(b.title))
-    .forEach((author) => {
-      // This updates the shared contributors object with meta information and is safe to be called multiple times.
-      author.url = path.join('/en', author.href);
-      author.data = {
-        title: author.title,
-        subhead: author.description,
-      };
-
-      author.elements = authorsPosts.has(author.key)
-        ? authorsPosts.get(author.key)
-        : [];
-
-      // If the author doesn't have any posts, use their Twitter profile.
-      if (author.elements.length === 0) {
-        if (!author.twitter) {
-          // If we're screenshot testing just ignore it.
-          if (process.env.PERCY) {
-            return;
-          }
-
-          // posts.length might be empty if we're generating partials so only
-          // log a warning if it has a value and we're doing a regular build.
-          // Note this is checking for _all_ posts and not just the posts
-          // by this specific author.
-          if (posts.length) {
-            console.warn(
-              `author ${
-                author.title
-              } has no posts and no social: ${JSON.stringify(author)}\n`,
-            );
-          }
-        } else {
-          author.href = `https://twitter.com/${author.twitter}`;
+  Object.keys(contributors).forEach((key) => {
+    const author = contributors[key];
+    author.key = key;
+
+    // Generate the contributor's name out of valid given/family parts. This
+    // allows our authors to just have a single name.
+    const parts = [author.name.given, author.name.family].filter(
+      (s) => s && s.length,
+    );
+    author.title = parts.join(' ');
+    author.href = `/authors/${key}/`;
+
+    author.description =
+      author.description && author.description.en
+        ? author.description.en
+        : `Our latest news, updates, and stories by ${author.title}.`;
+
+    // This updates the shared contributors object with meta information and is safe to be called multiple times.
+    author.url = path.join('/en', author.href);
```

Comment:
What's author.url being used for? I'm just cautious about hardcoding any language paths given @devnook's localization work.

Reply:
I believe it's being used by the BaseCard for [the url](https://github.com/GoogleChrome/web.dev/blob/master/src/site/_includes/components/BaseCard.js#L42).

Reply:
This can and would be removed when the cards are refactored.

Reply:
ah i see. sneaky old stripLanguage..